DICOMPlugins/DICOMTID1500Plugin.py

```diff
@@ -124,6 +148,114 @@ def createLoadableAndAddReferences(self, datasets):
+      # This only adds the particular SOPs that the bbox/lines are on. 
+      # if (loadable.ReferencedSegmentationInstanceUIDs[uid] == []):
+      #   loadable.referencedInstanceUIDs += loadable.ReferencedOtherInstanceUIDs
+      # We need to add all of the SOPs for a series 
+      if (loadable.ReferencedSegmentationInstanceUIDs[uid] == []): 
+        # Iterate through the loadable.ReferencedOtherInstanceUIDs, and get the associated SeriesInstanceUIDs 
+        SeriesInstanceUIDs = [] 
+        for SOPInstanceUID in loadable.ReferencedOtherInstanceUIDs: 
+          filePath = slicer.dicomDatabase.fileForInstance(SOPInstanceUID)
+          SeriesInstanceUIDs.append(slicer.dicomDatabase.fileValue(filePath, "0020,000E"))
+        # Then get a unique list of the SeriesInstanceUID, likely is 1
+        SeriesInstanceUIDs = list(set(SeriesInstanceUIDs))
+        # Now for each of these SeriesInstanceUIDs, get the list of SOPInstanceUIDs 
+        if (SeriesInstanceUIDs):
+          db = slicer.dicomDatabase
+          SOPInstanceUIDs = [] 
+          for SeriesInstanceUID in SeriesInstanceUIDs: 
+            fileList = db.filesForSeries(SeriesInstanceUID) 
+            for file in fileList: 
+              SOPInstanceUIDs.append(db.fileValue(file, "0008,0018"))
+          SOPInstanceUIDs = list(set(SOPInstanceUIDs))
+          # Now add to the loadable.referencedInstanceUIDs 
+          if (SOPInstanceUIDs):
+            loadable.referencedInstanceUIDs += SOPInstanceUIDs 
+
+      # # Now, we also check for PertinentOtherEvidenceSequence
+      # if (loadable.ReferencedSegmentationInstanceUIDs[uid] == []) and (loadable.referencedInstanceUIDs == []):
+      #   SeriesInstanceUIDs = [] 
+      #   if hasattr(dataset, "PertinentOtherEvidenceSequence"):
+      #     for refSeriesSequence in dataset.PertinentOtherEvidenceSequence:
+      #       for referencedSeriesSequence in refSeriesSequence.ReferencedSeriesSequence:
+      #         SeriesInstanceUIDs.append(referencedSeriesSequence.SeriesInstanceUID)
+      #   print('SeriesInstanceUIDs: ' + str(SeriesInstanceUIDs))
+      #   # Now for each of these SeriesInstanceUIDs, get the list of SOPInstanceUIDs 
+      #   if (SeriesInstanceUIDs):
+      #     db = slicer.dicomDatabase
+      #     SOPInstanceUIDs = [] 
+      #     for SeriesInstanceUID in SeriesInstanceUIDs: 
+      #       fileList = db.filesForSeries(SeriesInstanceUID) 
+      #       for file in fileList: 
+      #         SOPInstanceUIDs.append(db.fileValue(file, "0008,0018"))
+      #     SOPInstanceUIDs = list(set(SOPInstanceUIDs))
+      #     # Now add to the loadable.referencedInstanceUIDs 
+      #     if (SOPInstanceUIDs):
+      #       loadable.referencedInstanceUIDs += SOPInstanceUIDs 
+          
+        # We also check for FrameOfReferenceUID, and add to the list of loadable.referencedInstanceUIDs 
+        if (loadable.ReferencedSegmentationInstanceUIDs[uid] == []) and (loadable.referencedInstanceUIDs == []): 
```

Comment:
Since as I understand you are following @dclunie guidance on recognizing the specific pattern, I would reference here David's white paper http://dx.doi.org/10.13140/RG.2.2.34520.62725 and mention this briefly.

Reply:
I've added this information now. 

Fixed in this [branch](https://github.com/deepakri201/QuantitativeReporting/tree/add_planar_annot).